dvc/repo/experiments/executor/manager/base.py

```diff
@@ -183,7 +183,7 @@ def _exec_attached(self, repo: "Repo", jobs: Optional[int] = 1):
                     executor.reproduce,
                     info=executor.info,
                     rev=rev,
-                    queue=pid_q,
```

Comment:
mypy complains of wrong typing:

```
dvc/repo/experiments/executor/manager/base.py: note: In member "_exec_attached" of class "BaseExecutorManager":                           
dvc/repo/experiments/executor/manager/base.py:186:27: error: Argument "queue"                                                             
to "submit" of "Executor" has incompatible type "queue.Queue[Any]"; expected                                                              
"Optional[multiprocessing.queues.Queue[Any]]"  [arg-type]                                                                                 
                        queue=pid_q,                                                                                                      
                              ^                                                                                                           
Found 1 error in 1 file (checked 17 source files)            
```

Which doesn't seem correct looking at the fact that 
pid_q is  coming from`multiprocessing.Manager.Queue`:
https://github.com/python/cpython/blob/0e16105af69ec33e1215e0543f26a33bc8d10a16/Lib/multiprocessing/context.py#L102